concourse/build-frontend.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+cd npm-install-result
+npm run build
```

Comment:
Hier brauchst du `API_URL=https://spluseins.de npm run build`, weil sonst in den generierten JavaScripts localhost als Ziel steht.

Reply:
stimmt